plugins/pmsmart/src/test/java/org/itsallcode/whiterabbit/plugin/pmsmart/PMSmartExporterTest.java

```diff
@@ -71,6 +72,26 @@ void exportActivity()
         verify(projectRowMock).enterDuration(date, Duration.ofHours(1));
     }
 
+    @Test
+    void exportActivityWithoutComments()
+    {
+        when(configMock.getOptionalValue("transfer.comments")).thenReturn(Optional.of("false"));
+
+        final LocalDate date = LocalDate.of(2021, Month.MAY, 3);
+        runExport(day(date, DayType.WORK, activity(COST_CARRIER, Duration.ofHours(1), "project1")));
+        verify(projectRowMock).enterDuration(date, Duration.ofHours(1));
+        verify(projectRowMock, never()).enterComment(null, "never");
```

Comment:
```suggestion
        verify(projectRowMock, never()).enterComment(any(), any());
```
When using never(), better be more generic with the arguments to be sure the method is not called at all.
You might need to static import `Mockito.any()`